src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/Commands.GuestConfiguration.csproj

```diff
@@ -0,0 +1,116 @@
+    <CodeAnalysisRuleDirectories>;$(ProgramFiles)\Microsoft Visual Studio 12.0\Team Tools\Static Analysis Tools\FxCop\Rules</CodeAnalysisRuleDirectories>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
+    <DelaySign>true</DelaySign>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Cmdlets\GetAzureRmVMGuestPolicyReport.cs" />
+    <Compile Include="Common\Constants.cs" />
+    <Compile Include="Common\ParameterHelpMessages.cs" />
+    <Compile Include="Common\ParameterSetNames.cs" />
+    <Compile Include="Common\GuestConfigurationCmdletBase.cs" />
+    <Compile Include="Common\UserMessages.cs" />
+    <Compile Include="Helpers\CommonHelpers.cs" />
+    <Compile Include="Models\GuestConfigurationPolicyAssignment.cs" />
+    <Compile Include="Models\GuestConfigurationPolicyAssignmentReport.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="AzureRM.GuestConfiguration.psd1">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="help\Get-AzureRmVMGuestPolicyReport.md" />
+    <None Include="Microsoft.Azure.Commands.GuestConfiguration.format.ps1xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Include="MSSharedLibKey.snk" />
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\Profile\Commands.Profile\Commands.Profile.csproj">
+      <Project>{142d7b0b-388a-4ceb-a228-7f6d423c5c2e}</Project>
+      <Name>Commands.Profile</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Resources\Commands.ResourceManager\Cmdlets\Commands.Resources.Rest.csproj">
+      <Project>{e8ffea7f-a9cb-4fe6-a4d3-9334125b57b4}</Project>
+      <Name>Commands.Resources.Rest</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Resources\Commands.Resources\Commands.Resources.csproj">
+      <Project>{e1f5201d-6067-430e-b303-4e367652991b}</Project>
+      <Name>Commands.Resources</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.GuestConfiguration, Version=0.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.GuestConfiguration.0.9.0-preview\lib\net452\Microsoft.Azure.Management.GuestConfiguration.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Azure.Management.ResourceManager, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
You should not take dependencies on other clients in your cmdlet code.  If you need the resource management client for anything, you should use the 'Microsoft.Azure.PowerShell.ResourceManager' package